dwarf/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.6)
+
+project(dwarf)
+
+set(dwarfH ${CMAKE_CURRENT_LIST_DIR}/dwarf++.hh)
+set(dataH ${CMAKE_CURRENT_LIST_DIR}/data.hh)
+set(to_string_src ${CMAKE_CURRENT_BINARY_DIR}/to_string.cc)
+set(configured_to_string_script ${CMAKE_CURRENT_BINARY_DIR}/create_to_string_code.sh)
+
+configure_file(create_to_string_code.sh ${configured_to_string_script})
+
+add_custom_command(
+  OUTPUT ${to_string_src}
+  COMMAND bash ${configured_to_string_script} ${dwarfH} ${dataH} ${to_string_src})
```

Comment:
bash is not really cross platform (it is less often available on Windows then python).
I would just use a `add_custom_command` with all commands from the bash script.

See https://github.com/conan-io/conan-center-index/pull/3283 for an example.